extension/src/lambda_request_error.rs

```diff
@@ -166,46 +135,46 @@ mod tests {
 
   #[test]
   fn test_worse() {
-    use LambdaRequestError::*;
+    use LambdaRequestError as E;
```

Comment:
If an enum has many variants and you want to avoid typing the name repeatedly, it is safer to `use ... as` to ensure the problem above can't happen.